createst.py

```diff
@@ -143,8 +143,15 @@ def write_to_file(data):
         fp.write("# *** Add configuration here ***\n\n")
         if not args["strictcsums"]:
             fp.write("args:\n- -k none\n\n")
+        if check_requires():
+            fp.write("requires:\n")
+        if args["min_version"]:
+            fp.write("   min-version: %s\n\n" % args["min_version"])
         fp.write(data)
 
+def check_requires():
+    if args["min_version"]:
```

Comment:
Perhaps we should make a list of all the features that could be there in the requires section and check for the feature in that?
Something like:

features = ["min-version"]    # can have min-version for now
if any of the *features* are in *args*, write `requires` once to test.yaml

This will probably make it easier to add other features to `requires` section, it will just be a list updation and a write command.